pkgs/development/python-modules/opnieuw/default.nix

```
@@ -0,0 +1,28 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  typing-extensions,
+}:
+
+buildPythonPackage rec {
+  pname = "opnieuw";
+  version = "3.0.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-BxhSz+AsL7lU3zw9tIL2R+Pxktb8NG2/UKPtEDJT+Qo=";
+  };
+
+  propagatedBuildInputs = [ typing-extensions ];
+
+  meta = with lib; {
+    description = "Python retrying library: One weird trick to make your code more reliable";
+    homepage = "https://github.com/channable/opnieuw";
```

```suggestion
    homepage = "https://github.com/channable/opnieuw";
    changelog = "https://github.com/channable/opnieuw/blob/master/CHANGELOG.md";
```
Or link to the latest version if you prefer.